semgrep-core/parsing/Parse_kotlin_tree_sitter.ml

```diff
@@ -216,12 +216,12 @@ let as_operator (env : env) (x : CST.as_operator) =
 
 let function_modifier (env : env) (x : CST.function_modifier) =
   (match x with
-   | `Tail tok -> token env tok (* "tailrec" *)
-   | `Op tok -> token env tok (* "operator" *)
-   | `Infix tok -> token env tok (* "infix" *)
-   | `Inline tok -> token env tok (* "inline" *)
-   | `Exte tok -> token env tok (* "external" *)
-   | `Susp tok -> token env tok (* "suspend" *)
+   | `Tail tok -> NamedAttr (token env tok, [str env tok], empty_id_info (), fake_bracket []) (* "tailrec" *)
+   | `Op tok -> NamedAttr (token env tok, [str env tok], empty_id_info (), fake_bracket []) (* "operator" *)
+   | `Infix tok -> NamedAttr (token env tok, [str env tok], empty_id_info (), fake_bracket []) (* "infix" *)
+   | `Inline tok -> NamedAttr (token env tok, [str env tok], empty_id_info (), fake_bracket []) (* "inline" *)
```

Comment:
This one also maybe we could promote to a regular KeywordAttr.

Reply:
I think thats a good idea 👍